cpu/cortexm_common/include/core_cm0.h

```diff
@@ -1,13 +1,13 @@
 /**************************************************************************//**
  * @file     core_cm0.h
  * @brief    CMSIS Cortex-M0 Core Peripheral Access Layer Header File
- * @version  V4.00
- * @date     22. August 2014
+ * @version  V4.10
```

Comment:
the web page and zip archive says version 4.3, this particular header probably didn't change in the 4.2, 4.3 versions.